src/renderer/Default.js

```diff
@@ -45,13 +45,21 @@ import ModalsLayer from "./ModalsLayer";
 import { ToastOverlay } from "~/renderer/components/ToastOverlay";
 import Drawer from "~/renderer/drawers/Drawer";
 import UpdateBanner from "~/renderer/components/Updater/Banner";
+import useEnv from "~/renderer/hooks/useEnv";
 
 export const TopBannerContainer: ThemedComponent<{}> = styled.div`
   position: sticky;
   top: 0;
   z-index: 19;
 `;
 
+const SwapRoute = () => {
```

Comment:
My `/swap` vs `/swap2` approach was more for demonstration than anything else, I believe both options should go to `/swap` in order not to break all the other ways we can access the feature (action/currency-specific ctas), I suggest getting rid of the `SwapRoute` component and having a component that depends on that flag but a single Route

```
const SwapCmp = useMemo(() => (isSwapV2Enabled ? Swap2 : Swap), [isSwapV2Enabled]);
// Then further down
<Route path="/swap" render={props => <SwapCmp {...props} />} exact />
```

Reply:
To be honest, that was my initial idea but I was afraid that I missed some hidden subpaths (`/swap/:id`, `/swap/toto`) somewhere so I made the distinction to avoid any conflicts. I suppose that's because I'm not totally confident with the codebase yet. 

Will merge everything into the same `/swap` as you mentioned. Thanks for your review 👍